bluebottle/assignments/views.py

```diff
@@ -82,6 +83,17 @@ class ApplicantDetail(JsonApiViewMixin, AutoPrefetchMixin, RetrieveUpdateAPIView
         'document': ['document'],
     }
 
+    def perform_update(self, serializer):
```

Comment:
Don't we want to move this to the model, so that these transitions also happen in the back-office?

Reply:
Sounds good...